.travis.yml

```
@@ -0,0 +1,30 @@
+dist: bionic
+os: linux
+
+language: rust
+cache: cargo
+
+jobs:
+  fast_finish: true
+
+  include:
+    - name: "Stable"
+      rust: stable
+      script: 
+        - cargo build --verbose --all
+        - cargo test --verbose --all
+
+    - name: "Beta"
+      rust: beta
+      script:
+        - cargo build --verbose --all
+        - cargo test --verbose --all
+
+    - name: "Nightly"
+      rust: nightly
+      script: 
+        - cargo build --verbose --all
+        - cargo test --verbose --all
```

the script can be common:

```suggestion
  include:
    - name: "Stable"
      rust: stable
    - name: "Beta"
      rust: beta
    - name: "Nightly"
      rust: nightly
      
script:
  - cargo build --verbose --all
  - cargo test --verbose --all
```